src/react-image-lightbox.js

```diff
@@ -1614,6 +1615,8 @@ class ReactImageLightbox extends Component {
             </div>
           )}
         </div>
+
```

Comment:
Can you remove this new line?

Reply:
Looks like someone else resolved this edit over a year ago. By the way, this is a really small nitpick for a PR that has been open for over 2 years.